lib/media/drm_engine.js

```diff
@@ -1173,8 +1227,10 @@ shaka.media.DrmEngine = class {
     request.sessionId = session.sessionId;
     // NOTE: allowCrossSiteCredentials can be set in a request filter.
 
-    if (this.currentDrmInfo_.keySystem == 'com.microsoft.playready' ||
-        this.currentDrmInfo_.keySystem == 'com.chromecast.playready') {
+    if (
+      shaka.media.DrmEngine.isPlayReadyKeySystem_(
+          this.currentDrmInfo_.keySystem) ||
```

Comment:
I thought the new key system wouldn't require this workaround anymore.

Reply:
I've tested it and it still needs it.